db/migrate/20200618212422_add_subscribed_to_user_subscriptions_count_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddSubscribedToUserSubscriptionsCountToUsers < ActiveRecord::Migration[6.0]
+  def self.up
+    add_column :users, :subscribed_to_user_subscriptions_count, :integer, null: false, default: 0
+  end
+
+  def self.down
+    remove_column :users, :subscribed_to_user_subscriptions_count
+  end
+end
```

Comment:
Any reason not to make this a `def change` method? IIRC, using `add_column` in a `change` should be reversible and run `remove_column` by default, right?

Reply:
Yep, `change` would work! TBH I just used the built-in generator provided by the [counterculture gem](https://github.com/magnusvk/counter_culture) to create this migration. And since it's more explicit, I didn't bother changing it 🙈